src/concepts/backends/alternate-screen.md

```diff
@@ -11,6 +11,96 @@ other terminal content.
 This creates a seamless transition between the application and the regular terminal session, as the
 content displayed before launching the application will reappear after the application exits.
 
+Take this "hello world" program below. If we include the
+`std::io::stderr().execute(EnterAlternateScreen)?` (and the corresponding `LeaveAlternateScreen`),
+you can see how the program behaves differently.
+
+```rust
```

Comment:
The example still seems pretty heavy for the context (which is only the 2 lines that enter and leave the alternate screen.

Reply:
Not sure how to simplify it any further. I mainly want to show that we are rendering a simple `Paragraph` widget and the gif shows the difference with and without using `EnterAlternateScreen`.

Reply:
What about:

```rust
# use crossterm::{
#     terminal::{EnterAlternateScreen, LeaveAlternateScreen},
#     ExecutableCommand,
# };
# use ratatui::{prelude::*, widgets::*};
# use std::{
#     io::{stderr, Result},
#     thread::sleep,
#     time::Duration,
# };
#
# fn main() -> Result<()> {
let mut terminal = Terminal::new(CrosstermBackend::new(stderr()))?;
stderr().execute(EnterAlternateScreen)?;

terminal.draw(|f| {
    f.render_widget(Paragraph::new("Hello World!"), Rect::new(10, 10, 20, 1));
})?;
sleep(Duration::from_secs(2));

stderr().execute(LeaveAlternateScreen)?;
# Ok(())
# }
```

<img width="861" alt="image" src="https://github.com/ratatui-org/ratatui-book/assets/381361/62f3d185-fd67-4a11-99ec-efb9429f096e">

Reply:
woah, I don't think I knew about this hide lines feature! If I did read about it in the mdbook documentation, I promptly forgot about it.

Reply:
:)